diff --git a/node_modules/three-universal/examples/node-jsm/loaders/FBXLoader.js b/node_modules/three-universal/examples/node-jsm/loaders/FBXLoader.js
index 80a1d9e..d54b0bb 100644
--- a/node_modules/three-universal/examples/node-jsm/loaders/FBXLoader.js
+++ b/node_modules/three-universal/examples/node-jsm/loaders/FBXLoader.js
@@ -45,7 +45,7 @@ import {
 	URL,
 	window
 } from "../../../build/three.module.node.js";
-import * as fflate from '../libs/fflate.module.min.js';
+import * as fflate from '../../jsm/libs/fflate.module.min.js';
 import { NURBSCurve } from '../curves/NURBSCurve.js';
 
 /**
